src/middleware.js

```diff
@@ -37,11 +37,18 @@ function apiMiddleware({ getState }) {
 
       // Parse the validated RSAA action
       const callAPI = action[RSAA];
-      var { endpoint, body, headers, options = {} } = callAPI;
+      var {
+        endpoint,
+        body,
+        headers,
+        options = {},
+        fetch: customFetch
```

Comment:
Nice! For simplicity below (line 51), what if this line read: `fetch: doFetch = global.fetch`

This way, we can call `doFetch` internally and it will always be `callAPI.fetch`, with a default of `global.fetch`

Reply:
Yes, good call!

Reply:
Actually we can't always rely on `global` being defined right?

Reply:
Great catch. Yeah, this should be just `fetch`. This middleware expects fetch to be available globally, whether that's on `window` in browsers or `global` in node.

Reply:
Updated to default to `fetch`